### ISSUE TITLE
Add "how to ask for help" to wiki

### ISSUE THREAD

**Issue by khiron** [cogent3/Cogent3Workshop#16]

**Comment by khiron** [comment]
With cogent3workshop as a forkable repo, attendees could raise a pull request containing some code they have questions about that we can review and provide comments.

**Comment by GavinHuttley** [comment]
I don't think this is the right mechanism. We want to reinforce their usage of standard GitHub repo tools for raising issues and the process for doing that:
- how to make a minimal algorithm and minimal data
- raising a topic on GitHub discussions, embedding code using the syntax gates, or as an attachment OR raising a GitHub issue if it's a bug or feature request
